captum/insights/config.py

```diff
@@ -33,20 +43,47 @@ class StrEnumConfig(NamedTuple):
     IntegratedGradients,
     Saliency,
     FeatureAblation,
+    Occlusion,
 ]
 
+
+class ConfigParameters(NamedTuple):
+    params: Dict[str, NamedTuple]
+    help_info: Optional[str] = None  # TODO fill out help for each method
+    post_process: Optional[Callable[[Any], Any]] = None
+
+
 ATTRIBUTION_NAMES_TO_METHODS = {
     # mypy bug - treating it as a type instead of a class
     cls.get_name(): cls  # type: ignore
     for cls in SUPPORTED_ATTRIBUTION_METHODS
 }
 
+
+def _str_to_tuple(s):
+    if isinstance(s, tuple):
+        return s
+    return tuple([int(i) for i in s.split()])
+
+
 ATTRIBUTION_METHOD_CONFIG: Dict[str, Dict[str, tuple]] = {
-    IntegratedGradients.get_name(): {
-        "n_steps": NumberConfig(value=25, limit=(2, None)),
-        "method": StrEnumConfig(limit=SUPPORTED_METHODS, value="gausslegendre"),
-    },
-    FeatureAblation.get_name(): {
-        "perturbations_per_eval": NumberConfig(value=1, limit=(1, 100)),
-    },
+    IntegratedGradients.get_name(): ConfigParameters(
+        params={
+            "n_steps": NumberConfig(value=25, limit=(2, None)),
+            "method": StrEnumConfig(limit=SUPPORTED_METHODS, value="gausslegendre"),
+        }
+    ),
+    FeatureAblation.get_name(): ConfigParameters(
+        params={"perturbations_per_eval": NumberConfig(value=1, limit=(1, 100)),}
+    ),
+    Occlusion.get_name(): ConfigParameters(
+        params={
+            "sliding_window_shapes": StrConfig(value=""),
+            "strides": StrConfig(value=""),
+        },
+        post_process={
```

Comment:
Why are these post-processing steps necessary?

Reply:
Occlusion takes in a tuple or an int e.g. `(3, 5, 5)`, so we need a way to convert a string representation into the appropriate format.